lib/modules/manager/helmfile/extract.ts

```diff
@@ -118,12 +117,12 @@ export async function extractPackageFile(
 
       if (isOCIRegistry(dep.chart)) {
         res.datasource = DockerDatasource.id;
-        res.packageName = joinUrlParts(repoName, depName);
+        res.packageName = repoName + '/' + depName;
```

Comment:
I'm pretty sure we do this all over the place for docker references 🤔 

Have you checked that?  
In that case this should be new utility and should not be part of manager logic

Reply:
I'm just rolling back to how it was before i changed it and caused it to fail,
If you would like to open an issue to refactor later be my guest.
for now let's make our user happy and solve this.

Reply:
```suggestion
        res.packageName = `${repoName}/${depName}`;
```
use string template